### PR TITLE
Add jsonresume-theme-relaxed

### PR DESCRIPTION
**Pull request by ObserverOfTime**
Closes jsonresume/registry-functions#72

## Summary by CodeRabbit

- **New Features**
  - Introduced a new 'relaxed' theme for enhanced visual options.

- **Refactor**
  - Reorganized theme options for better accessibility and consistency.

- **Style**
  - Standardized comment formatting within theme configuration for improved code readability.

### CODE REVIEW

File: apps/registry/pages/api/formatters/template.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ export const THEMES = {
   dinesh: require('jsonresume-theme-dinesh'),
   elegant: require('jsonresume-theme-elegant'),
   elite: require('jsonresume-theme-elite'),
-  // eloquent: require("jsonresume-theme-eloquent"),
+  // eloquent: require('jsonresume-theme-eloquent'),
```
[the remark this answers is not in the document]
Comment (ObserverOfTime) [reply apps/registry/pages/api/formatters/template.js:17]: I just noticed the inconsistencies and fixed them. I can revert them if you want.

[the remark this answers is not in the document]
Comment (levino) [reply apps/registry/pages/api/formatters/template.js:17]: I would prefer if you could explain which tool did these changes. Maybe we can add it to our linter / formatting rules.

**Review by levino — CHANGES_REQUESTED** [review]
Leaving this as a veto to merge. It looks to me as if these changes completely break the [registry](https://jsonresume-org-registry-git-fork-observerofti-fca28d-jsonresume.vercel.app/thomasdavis) (should return 200, not 500). Maybe this has to do with some kind of file system access that your implementation requires @ObserverOfTime. What is most concerning to me is, that we do not have a check that fails in these cases. Will have a look how I can add a test to immediately raise a flag in these cases.

@ObserverOfTime did you successfully locally run some tests with your theme? Which tests did you run?

File: apps/registry/pages/api/formatters/template.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (ObserverOfTime) [reply apps/registry/pages/api/formatters/template.js:17]: The best tool: a human (i.e. me).

**Comment by ObserverOfTime** [comment]
I see the issue. My theme expects the skill level to be one of [these](https://github.com/jsonresume/resume-cli/issues/213#issuecomment-234864045) but the sample file has 'Senior'. I'll publish a new version that supports it. Would it be possible to standardise skill levels in the future?

**Comment by ObserverOfTime** [comment]
I tested it by running `pnpm dev --filter=registry` and visiting http://localhost:3002/thomasdavis?theme=relaxed.
This results in `Error: Unexpected skill level: Senior` which means the issue is on my side.
I was making some other changes anyway so I'll publish a new version and see if that fixes it.

**Comment by levino** [comment]
I think you rather mean `pnpm --filter=registry dev`. However the page crashes with a problem with the `apage` theme for me (it tries to access `process.env.LANG`) which does not exist.

**Comment by ObserverOfTime** [comment]
> I think you rather mean `pnpm --filter=registry dev`.

That's not what the readme says.

> (it tries to access `process.env.LANG`) which does not exist.

Just set `LANG=C`.

---

Anyway, I've fixed it.

File: apps/registry/pages/api/formatters/template.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (levino) [reply apps/registry/pages/api/formatters/template.js:17]: I see :) Cannot replicate that in code unfortunately.

**Comment by levino** [comment]
> > I think you rather mean `pnpm --filter=registry dev`.
> 
> That's not what the readme says.

I ran the command from `apps/registry` and there it only works the other way round. My bad, sorry. Please also see #26 for details.

> > (it tries to access `process.env.LANG`) which does not exist.
> 
> Just set ~`LANG=C`~ `LANG=en`.

Even though this might be a workaround, I find it quite problematic to rely on a variable that is only set automatically in certain environments (e.g. specific linux distros) and would rather [remove](https://github.com/jsonresume/jsonresume.org/pull/27) the theme.

> Anyway, I've fixed it.

Waiting for #25, then I will update your branch and then we can go ahead.

**Review by levino — APPROVED** [review]
I take the liberty to thank you in the name of @thomasdavis for your valuable contribution, @ObserverOfTime

**Review by levino — CHANGES_REQUESTED** [review]
Please rebase on `master` if you find the time. The deployment of the registry does not work. Whenever I try to GET `/thomasdavis` the code crashes with:
```
Error: stylus:280:17
   276|       float right
   277| 
   278|   html[dir="rtl"]
   279|     #image > img, .float
   280|       float left
------------------------^
   281| 

failed to locate @import file /var/task/node_modules/.pnpm/stylus@0.62.0/node_modules/stylus/lib/functions/index.styl

    at Evaluator.visitImport (/var/task/node_modules/.pnpm/stylus@0.62.0/node_modules/stylus/lib/visitor/evaluator.js:911:23)
    at Evaluator.visit (/var/task/node_modules/.pnpm/stylus@0.62.0/node_modules/stylus/lib/visitor/index.js:29:42)
    at Evaluator.visit (/var/task/node_modules/.pnpm/stylus@0.62.0/node_modules/stylus/lib/visitor/evaluator.js:154:20)
    at Evaluator.visitRoot (/var/task/node_modules/.pnpm/stylus@0.62.0/node_modules/stylus/lib/visitor/evaluator.js:703:29)
    at Evaluator.visit (/var/task/node_modules/.pnpm/stylus@0.62.0/node_modules/stylus/lib/visitor/index.js:29:42)
    at Evaluator.visit (/var/task/node_modules/.pnpm/stylus@0.62.0/node_modules/stylus/lib/visitor/evaluator.js:154:20)
    at Evaluator.evaluate (/var/task/node_modules/.pnpm/stylus@0.62.0/node_modules/stylus/lib/visitor/evaluator.js:241:17)
    at Renderer.render (/var/task/node_modules/.pnpm/stylus@0.62.0/node_modules/stylus/lib/renderer.js:70:28)
    at exports.render (/var/task/node_modules/.pnpm/stylus@0.62.0/node_modules/stylus/lib/stylus.js:82:37)
    at Object.<anonymous> (/var/task/node_modules/.pnpm/jsonresume-theme-relaxed@0.1.5/node_modules/jsonresume-theme-relaxed/index.js:121:22)
    at Module._compile (node:internal/modules/cjs/loader:1256:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1310:10)
    at Module.load (node:internal/modules/cjs/loader:1119:32)
    at Module._load (node:internal/modules/cjs/loader:960:12)
    at r.<computed>.e._load (/var/task/___vc/__launcher/__launcher.js:14:2516)
    at Module.require (node:internal/modules/cjs/loader:1143:19) {
  lineno: 280,
  column: 17,
  filename: 'stylus',
  stylusStack: ''
}
Error: Runtime exited with error: exit status 1
Runtime.ExitError
```
I cannot reproduce this error locally. If you can, please let me know how. Thank you. Maybe it has to do with running this code serverless / as a lambda function, which does not make it easier to debug, I fear.

**Comment by levino** [comment]
I would assume that the [`fs.readFile`](https://github.com/ObserverOfTime/jsonresume-theme-relaxed/blob/ad35001b410a2a0eb83eb56fe8e663242c7a25a9/index.js#L121) (sync or not sync) is the problem in a serverless environment since the code propably gets bundled and only files that are correctly `import`ed will be bundled with. @thomasdavis said something about problems for handlebars and reading from the file system as well, iirc.

**Comment by ObserverOfTime** [comment]
> Even though this might be a workaround, I find it quite problematic to rely on a variable that is only set automatically in certain environments (e.g. specific linux distros)

It seems to be used by the theme for i18n though.

> I cannot reproduce this error locally.

I'll look into it again tomorrow but I haven't encountered it so far.

**Comment by ObserverOfTime** [comment]
I cannot reproduce it locally but it shouldn't be an issue with `fs.readFile` since the other usages work.
It's most likely an unresolved stylus issue (stylus/stylus#2711).

**Comment by ObserverOfTime** [comment]
I published a new version that precompiles the stylesheet. Hopefully it'll work now.

**Comment by thomasdavis** [comment]
It still is throwing a 500 -> https://jsonresume-org-registry-4wi0nemyq-jsonresume.vercel.app/thomasdavis?theme=relaxed

```
Error: stylus:280:17
   276|       float right
   277| 
   278|   html[dir="rtl"]
   279|     #image > img, .float
   280|       float left
------------------------^
   281| 

failed to locate @import file /var/task/node_modules/.pnpm/stylus@0.62.0/node_modules/stylus/lib/functions/index.styl

    at Evaluator.visitImport (/var/task/node_modules/.pnpm/stylus@0.62.0/node_modules/stylus/lib/visitor/evaluator.js:911:23)
```

Will have a look too

**Comment by ObserverOfTime** [comment]
Weird, it shouldn't be using stylus anymore. :thinking:

**Comment by ObserverOfTime** [comment]
Looks like it works now. :tada:

**Comment by levino** [comment]
It does. @thomasdavis I think you forgot to approve the redeployment of the preview of the registry after @ObserverOfTime pushed their changes with the new version `1.0.6` of their theme. I approved the deployment with the new changes and it is working indeed. Merging.

**Review by levino — APPROVED** [review]

**Pull request merged**